pkg/crinit/aggregated/init.go

```diff
@@ -199,27 +199,15 @@ func createRBACObjects(cmdOut io.Writer, clientset client.Interface,
 
 	glog.V(4).Info("Successfully created service account")
 
-	// Create a Kubernetes cluster role to allow REST operations on our
-	// cluster registry API resources e.g. Cluster.
-	glog.V(4).Infof("Creating cluster role %v", clusterRoleName)
-
-	cr, err := createClusterRole(clientset, opts.DryRun)
-
-	if err != nil {
-		glog.V(4).Infof("Failed to create cluster role %v: %v", cr, err)
-		return nil, err
-	}
-
-	glog.V(4).Info("Successfully created cluster role")
-
 	// Create a Kubernetes cluster role binding from the default service account
-	// in our namespace to the cluster role we just created.
-	glog.V(4).Infof("Creating cluster role bindings %v and %v", apiServerCRBName, authDelegatorCRBName)
+	// in our namespace to the system:auth-delegator cluster role.
```

Comment:
The `system:auth-delegator` cluster role is part of the default RBAC roles described [here](https://kubernetes.io/docs/admin/authorization/rbac/#default-roles-and-role-bindings). A quick look through the code shows that these default RBAC cluster roles are created by the `kube-apiserver` as part of the [`rbac/bootstrap-roles`](https://github.com/kubernetes/kubernetes/blob/07240b7166d83bed49d783e0ecdfa7ee7e62cfca/pkg/registry/rbac/rest/storage_rbac.go#L60) [`PostStartHook`](https://github.com/kubernetes/kubernetes/blob/07240b7166d83bed49d783e0ecdfa7ee7e62cfca/pkg/registry/rbac/rest/storage_rbac.go#L120) for the [RBAC REST storage provider](https://github.com/kubernetes/kubernetes/blob/07240b7166d83bed49d783e0ecdfa7ee7e62cfca/pkg/master/master.go#L347:12).

Based on that code and the doc linked above, unless somebody tampers with the defaults, I think this is expected to exist in all clusters that support and enable RBAC (`--authorization-mode=RBAC`). I think we can assume or at least expect that RBAC will be enabled. Otherwise, the entire cluster registry aggregated deployment will fail. But it probably wouldn't hurt to add a check for RBAC enablement with a graceful error if it's disabled.